src/App.js

```diff
@@ -1,7 +1,36 @@
 import "./App.css";
+import ArticleCard from "./components/ArticleCard";
+import Articles from "./components/Articles";
+import Comments from "./components/Comments";
+import Footer from "./components/Footer";
+import Header from "./components/Header";
+import PostAComment from "./components/PostAComment";
+import SelectAuthor from "./components/SelectAuthor";
+import SelectTopic from "./components/SelectTopic";
+import Users from "./components/Users";
+import { Route, Routes } from "react-router-dom";
+import { useParams } from "react-router-dom";
 
 function App() {
-  return <div className="App"></div>;
+  return (
+    <div className="App">
+      <Header />
+      <div className="Nav">
+        <SelectAuthor />
+        <SelectTopic />
+      </div>
+      <Routes>
+        <Route path="/" element={<Articles />} />
+        <Route path="/articles" element={<Articles />} />
+        <Route path="/articles/:article_id" element={<ArticleCard />} />
+      </Routes>
+      {/* <Users />
```

Comment:
I appreciate you've commented these out because you don't need them yet, but we shouldn't have any extra code that isn't relevant to the ticket you're working on. Best to delete these until you actually need them.